scripts/setup_build_env.sh

```diff
@@ -82,7 +82,7 @@ function get_and_patch_intel_drivers()
         error "MTL patch for ICE=v${ICE_VER} could not be found: ${MTL_DIR}/patches/ice_drv/${ICE_VER}"
         return 1
     fi
-    wget_download_strip_unpack "https://downloadmirror.intel.com/${IRDMA_DMID}/irdma-${IRDMA_VER}.tgz" "${IRDMA_DIR}"
+    NO_PROXY="" no_proxy="" wget_download_strip_unpack "https://downloadmirror.intel.com/${IRDMA_DMID}/irdma-${IRDMA_VER}.tgz" "${IRDMA_DIR}"
```

Comment:
1. Why `no_proxy` added to the script? I use it as well and I pass it to the script itself, but I think some users would like to use their own proxy setup.
2. Please move installation of irdma after the ice driver installation. (to line 88)